app/views/orga/teams/show.html.slim

```diff
@@ -74,12 +74,11 @@ nav.btn-group
   h3 Supervisors
   p = @team.supervisors.any? ? link_to_team_members(@team, :supervisor) : content_tag(:em, 'Will be added by RGSoC Organizers')
 
-- if @team.members.any? { |user| user.conferences.in_current_season.any? }
+- if @team.conferences.in_current_season.any?
   h4 Conferences
```

Comment:
If I recall correctly, for the if-condition to actually work, these two lines (the h4 and ul) need to be indented one more level.